images/airflow/2.9.2/python/mwaa/entrypoint.py

```diff
@@ -548,9 +538,51 @@ def on_sigterm() -> None:
                 on_sigterm=on_sigterm,
             )
         ]
-    )
 
 
+def _get_airflow_scheduler_subprocesses(environ: Dict[str, str], conditions: List):
+    """
+    Get the scheduler subproceses: scheduler, dag-processor, and triggerer.
+    
+    :param environ: A dictionary containing the environment variables.
+    :param conditions: A list of subprocess conditions.
+    :returns: Scheduler subprocesses.
+    """
+    return [
+            create_airflow_subprocess(
+                [airflow_cmd],
+                environ=environ,
+                logger_name=logger_name,
+                friendly_name=friendly_name,
+                conditions=conditions if airflow_cmd == "scheduler" else [],
+            )
+            for airflow_cmd, logger_name, friendly_name in [
+                ("scheduler", SCHEDULER_LOGGER_NAME, "scheduler"),
+                ("dag-processor", DAG_PROCESSOR_LOGGER_NAME, "dag-processor"),
+                ("triggerer", TRIGGERER_LOGGER_NAME, "triggerer"),
+            ]
+        ]
+
+
+def _get_conditions(airflow_cmd: str):
```

Comment:
`_get_conditions` is an ambiguous name. What conditions are we getting? I would rather use a long, slightly ugly, but informative name like `get_scheduler_worker_common_conditions` than an ambiguous name. Ideally though, an informative and beautiful name would be great.

Reply:
I think maybe _get_airflow_conditions would be better. I don't think the name requires the addition of "worker" or "scheduler" since this is essentially a parameter to the function. For the given airflow process command, this function returns the conditions required.

Reply:
`_get_airflow_conditions` is still vague in my opinion. Perhaps `_get_airflow_process_conditions` is better, since it becomes clearer what the conditions are for, and the parameter explains what Airflow process we are talking about.